app/controllers/chat_channel_memberships_controller.rb

```diff
@@ -138,7 +134,7 @@ def request_details
     user_chat_channels = ChatChannel.includes(:chat_channel_memberships).where(
       chat_channel_memberships: { user_id: current_user.id, role: "mod", status: "active" },
     )
-    @memberships = user_chat_channels.map(&:requested_memberships).flatten
+    @memberships = user_chat_channels.map(&:requested_memberships)&.flatten
```

Comment:
Ruby has a combination of `map` + `flatten` called `flat_map`:

```suggestion
    @memberships = user_chat_channels.flat_map(&:requested_memberships)
```